### PR TITLE
Add tracks events to Store Stats

### PR DESCRIPTION
**Pull request by psealock**
### Trigger analytics' recordPageView 
Results in `calypso_page_view` event in tracks.  ~**This initial commit has a placeholder string**~. This pull request can be used to track conversations around the events needed.

Resources
https://wpcalypso.wordpress.com/devdocs/client/lib/analytics/README.md?term=tracks

cc @greenafrican @westi

### CODE REVIEW

**Comment by psealock** [comment]
Fixes https://github.com/Automattic/wp-calypso/issues/16169

**Comment by greenafrican** [comment]
Looks good. I know I hadn't fleshed out the description of the issue but wondering if we need to track param changes and Google Analytics click events, for example:
https://github.com/Automattic/wp-calypso/blob/a70587157b33783ba4d5e0e4a1e136680d459d51/client/my-sites/stats/stats-list/action-follow.jsx#L36

**Comment by psealock** [comment]
>track param changes and Google Analytics click events

Every click/navigation changes the url which triggers the `analytics.pageView` function (records in GA and Tracks) with the exception of clicking tabs on the main chart.

It seems to me we should stick to Tracks events instead of Google Analytics. From internal description:

>However, because of our volume our data is heavily sampled. Additionally, GA is a per-session model and does not allow analysis across a user’s entire lifecycle.

**Comment by psealock** [comment]
### Added
Tracks event `calypso_store_stats_chart_tab_navigation` on tab clicks for main chart. This is the only user event not reflected in `pageView`.

This establishes a pattern `calypso_store_stats_*` for this project.

**Comment by psealock** [comment]
Changed to `calypso_woocommerce_stats_chart_tab_click` to match the pattern already present

File: client/extensions/woocommerce/app/store-stats/constants.js [hunk 1]
```diff
@@ -115,3 +115,10 @@ export const UNITS = {
 		title: translate( 'Years' )
 	}
 };
+
+export const chartTabs = [
+	{ label: 'Gross Sales', attr: 'gross_sales', type: 'currency' },
```
Comment (greenafrican) [inline client/extensions/woocommerce/app/store-stats/constants.js]: Translate perhaps?

File: client/extensions/woocommerce/app/store-stats/constants.js [hunk 2]
```diff
@@ -115,3 +115,10 @@ export const UNITS = {
 		title: translate( 'Years' )
 	}
 };
+
+export const chartTabs = [
+	{ label: translate( 'Gross Sales' ), attr: 'gross_sales', type: 'currency' },
+	{ label: translate( 'Net Sales' ), attr: 'net_sales', type: 'currency' },
+	{ label: translate( 'Orders' ), attr: 'orders', type: 'number' },
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/store-stats/constants.js]: Hi! I've found a possible matching string that has already been translated **100** times:
`translate( 'Order' )` *ES Score: 6*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/WCS-stats-analytics)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Pull request merged**